### DISCUSSION TITLE
Welcome to activemq-artemis-operator Discussions!

### DISCUSSION THREAD

**Discussion by brusdev** [arkmq-org/arkmq-org-broker-operator#399] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prateekjainaa** [comment]


**Reply by brusdev** [reply]
@prateekjainaa this discussion is only for welcome, could you create a new discussion and delete this comment?
